safe-global/web-core
# feat: page headers

## What it solves

Add page headers

## How this PR fixes it

The app-wide page headers have been added, as shown [here](https://5afe.slack.com/archives/C03DAGWJCR5/p1663773208255179).

`PageLayout`'s `.container main` styling adds the default padding for pages and the `main` element have therefore been moved outside of the page headers, sometimes into the page components themselves.

No extensive style adjustments have been made in order to keep this PR concise.

## How to test it

The following pages should have interactable (sticky) headers:

- Assets: tabs and currency selector
- Transactions: tabs and working batch execute/filter buttons respectively (the open filter should stick below the header on long lists)
- Address book: import, export and entry creation buttons (search function will be implemented later)
- Safe Apps: as before
- Settings: tabs

Safe Apps queue bar should retain batch execute functionality.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/191974488-e5ba7f4c-a830-4a6d-983d-58260bb83138.png)

![image](https://user-images.githubusercontent.com/20442784/191974648-dfa324ea-5037-44a2-b810-f2dabc16f521.png)

![image](https://user-images.githubusercontent.com/20442784/191974677-d0bbe400-5a26-41f7-af3e-a44754ab192d.png)

![image](https://user-images.githubusercontent.com/20442784/191974722-4e927b77-429a-4ee2-a16d-22985a2359cf.png)

![image](https://user-images.githubusercontent.com/20442784/191974763-a4d51e9d-758b-4081-acec-a46f5e9d80b7.png)

![image](https://user-images.githubusercontent.com/20442784/191974795-4ddcf130-734a-4c8f-9d06-9509aa0d330b.png)
